apps/service_providers/tests/test_runnables.py

```diff
@@ -66,9 +67,22 @@ def runnable(request, session):
     return runnables[request.param]
 
 
+def _get_history_manager(session, experiment=None):
+    return ExperimentHistoryManager.for_llm_chat(
+        session=session,
+        experiment=experiment if experiment else session.experiment,
+        max_token_limit=session.experiment.max_token_limit,
+        chat_model=session.experiment.get_chat_model(),
```

Comment:
seems like you could push these into the `for_llm_chat` method which would make it more similar to `ChatAdapter.for_experiment`